ViewGeneratorCore/build/ViewGeneratorCore.targets

```diff
@@ -24,16 +24,16 @@
       <ItemGroup Condition="'$(OS)' == 'Windows_NT'">
           <_ViewGeneratorCoreNodeJsExe Include="$(MSBuildThisFileDirectory)..\..\..\Node.js.redist\$(NodeVersion)\tools\win-x64\node.exe"/>
           <_ViewGeneratorCoreNodeJsExe Include="$(MSBuildThisFileDirectory)..\..\Node.js.redist.$(NodeVersion)\tools\win-x64\node.exe"/>
-
       </ItemGroup>
       <ItemGroup Condition="'$(OS)' != 'Windows_NT'">
           <_ViewGeneratorCoreNodeJsExe Include="$(MSBuildThisFileDirectory)..\..\..\Node.js.redist\$(NodeVersion)\tools\osx-x64\node"/>
           <_ViewGeneratorCoreNodeJsExe Include="$(MSBuildThisFileDirectory)..\..\Node.js.redist.$(NodeVersion)\tools\osx-x64\node"/>
-
       </ItemGroup>
-      
+      <ItemGroup>
+          <_ViewGeneratorCoreNodeJsExeAfterCheckedPaths Include="@(_ViewGeneratorCoreNodeJsExe)" Condition="Exists('%(FullPath)')"  />
+      </ItemGroup>
     <PropertyGroup>
-      <__ViewGeneratorCoreNodeJsExe>@(_ViewGeneratorCoreNodeJsExe);</__ViewGeneratorCoreNodeJsExe>
+      <__ViewGeneratorCoreNodeJsExe>@(_ViewGeneratorCoreNodeJsExeAfterCheckedPaths);</__ViewGeneratorCoreNodeJsExe>
```

Comment:
You need to do a substring right? You can have both file existing in filesystem

Reply:
We are still doing that in the line below with **__ViewGeneratorCoreNodeJsExe.Substring**

Reply:
Oh... Weird pattern, we're declaring the same property twice? Does msbuild complain about that?

Reply:
Right, I was confused with the variable naming